### ISSUE TITLE
Enabling virtual reality (i.e. Cardboard) destroys FPS

### ISSUE THREAD

**Issue by alexbu92** [google-ar/arcore-unity-sdk#102]
After using the workaround to be able to build with virtual reality support enabled alongside ARCore (found here: #94 ), I profiled the build and discovered that performance was significantly affected. From a smooth 33 ms (most of which spent waiting for the next image from the camera), it went down to 66 ms after enabling Cardboard with almost all of it spent in EarlyUpdate.TangoUpdate. What could be causing this?

**Comment by JP-Dorman** [comment]
Is it not just a case of having to render the scene twice (once for each eye) in VR mode, therefore the halving the frame rate?

**Comment by alexbu92** [comment]
@JP-Dorman I don't believe so. The time isn't actually doubling because as I said, most of the time to render a frame with only ARCore active is passed with the CPU in idle (analogous as to what happens when you enable v-sync on an otherwise capable of high fps machine for example). When Cardboard is also activated on the other hand, that idle time disappears completely and furthermore almost all of the time spent rendering a frame falls under this "EarlyUpdate.TangoUpdate" which is quite strange to me. 

I know Cardboard is quite taxing already on its own, but there's something that just doesn't seem right about the profiling log of ARCore vs ARCore + Cardboard.

**Comment by pablisho** [comment]
Thanks for the bug report. I added it to our internal tracking.
